test-suite/coq-makefile/timing/precomputed-time-tests/003-non-utf8/run.sh

```diff
@@ -5,20 +5,14 @@ set -e
 
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
-python2 "$COQLIB"/tools/make-one-time-file.py time-of-build.log.in time-of-build-pretty.log2 || exit $?
```

Comment:
Please don't.  I'd like to keep these files compatible with both python2 and python3.  Feel free to makes this test pass with a warning when python2 does not exist (but please continue installing it on the CI for this test).

Reply:
I'm not sure I see a good reason we should keep supporting `python2` in 8.12, anyways that discussion should be done in #10491